internal/dockerutil/setup.go

```diff
@@ -92,6 +92,7 @@ func DockerSetup(t DockerSetupTestingT) (*client.Client, string) {
 func dockerCleanup(t DockerSetupTestingT, cli *client.Client) func() {
 	return func() {
 		showContainerLogs := os.Getenv("SHOW_CONTAINER_LOGS") != ""
```

Comment:
Note: these env variables should be included in #673 docs for ictest

Reply:
Yes. Good call out. I started a doc for env vars in #725